### PR TITLE
Creeper vines show under things instead of over.

### LINKED ISSUE #23016: Creeper suggestion
Nanotrasen hire this man
Screenshot_173

### PR DESCRIPTION
**Pull request by Allspice**
Closes #23016

I tried to test it but I couldn't varedit on my local server for some reason.
:cl:
* tweak: Creeper vines show under things instead of over now.

### CODE REVIEW

**Comment by coldud13** [comment]
how hard is it to see the vines now if it's under a table or a console?

**Comment by Allspice** [comment]
depends if you look at the edges of the console/table
But to be clear, this affects creeper vines specifically instead of kudzu, which have a limit of 5? radius spread.

**Comment by DrSnips** [comment]
Vines are extremely annoying to deal with right now. Not sure how I feel about making them harder to cut.

**Comment by Killette2** [comment]
Lucky for you, creeper vines only grow up to a radius of 5 tiles from the center, forming a 9x9 grid at worst. Worst case scenario is the tray is being dragged around to create more of it

**Comment by DrSnips** [comment]
That doesn't make cleaning that 5 by 5 to 9 by 9 area any better.

File: code/modules/hydroponics/spreading/spreading.dm [hunk 1]
```diff
@@ -67,6 +67,7 @@
 	max_health = round(seed.endurance/2)
 	if(seed.spread == 1)
 		limited_growth = 1
+		plane = CREEPER_LAYER
```
Comment (I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE) [inline code/modules/hydroponics/spreading/spreading.dm]: >plane = X_LAYER
uhhhhhhhh

Comment (Allspice) [reply code/modules/hydroponics/spreading/spreading.dm]: "layer = " then, I'm guessing? vv wasn't working right on my end to check, but I can test on the test server later.

**Comment by SonixApache** [comment]
does this still need fixing @Allspice @Kurfursten

**Comment by Allspice** [comment]
The new commit works, yeah.

**Pull request merged**